build.default.props

```diff
@@ -400,3 +411,33 @@ varnish.server.port = 8888
 # Drush Context configuration.
 # ----------------------------
 drush.db.dump = ${build.platform.dir}/dump.sql
+
+# Drush aliases configuration.
+# ----------------------------
+drush.alias.default = ${build.site}
+drush.alias.docker = docker
+
+# Drupal installation settings.
+# ----------------------------
+drupal.update.free.access = FALSE
+drupal.hash.salt =
+
+# Bactrack configuration
+# ----------------------------
+backtrac.file = ${project.tmp.dir}/bactrack.json
+
+
+# Toolkit location: ./includes/phing/props/drone.props
+# ------------------------------------------------------------------------------
+# Default properties for toolkit pipeline on Drone:
+# https://github.com/ec-europa/ne-pipelines/blob/master/drone/toolkit.yml
+# ------------------------------------------------------------------------------
+
+db.host = mysql
+solr.host = solr
+behat.base_url = http://web:8080
```

Comment:
Duplicated and conflicting prop. See line 124 [here](https://github.com/ec-europa/toolkit/pull/85/files#diff-227ef8d635c62aa4600ecc332c8f22a2R124)

Reply:
Idem, above